packages/components/src/combobox-control/README.md

```diff
@@ -116,6 +116,13 @@ The current value of the input.
 -   Type: `mixed`
 -   Required: Yes
 
+#### looseSearch
+
+If present, search matching is made less restrictive by removing spaces and special characters.
```

Comment:
I would recommend we should be explicit here by telling what all the special characters we're talking about. 

What do you think?

Reply:
Done!